.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+indent_size = 2
+
+[*.go]
+indent_style = tab
+
+[Makefile]
```

Comment:
I get the feeling like if I was vibecoding haha, AIs always add extra things that are not needed. We don't have Makefile in the repo, this can be removed imho.